docs/ensemble_scheduler.md

```diff
@@ -25,7 +25,7 @@ This document presents ensemble models as an example of [DAG Scheduler](dag_sche
 ```
 ~$ virtualenv -p python3 .tf_env
 ~$ source .tf_env/bin/activate
-~$ pip3 install tensorflow==1.15
+~$ pip3 install tensorflow==2.3.1
```

Comment:
Have you tested the script manually? As far I remember `argmax_sum.py` requires tensorflow 1.15.